Graphics/GraphicsEngineOpenGL/src/RenderDeviceGLImpl.cpp

```diff
@@ -1687,11 +1687,23 @@ void RenderDeviceGLImpl::PurgeContextCaches(GLContext::NativeGLContextType Conte
 {
     {
         Threading::SpinLockGuard FBOCacheGuard{m_FBOCacheLock};
-        m_FBOCache.erase(Context);
+
+        auto it = m_FBOCache.find(Context);
+        if (it != m_FBOCache.end())
+        {
+            it->second.Clear();
```

Comment:
Why do you need to clear the cache before erasing it?
When the cache is destroyed all objects should be released anyway.